test/smokes/smoke.rb

```diff
@@ -109,11 +110,16 @@ def run
 
       def run_test(params, out)
         command_output, _ = Dir.mktmpdir do |dir|
-          repo_dir, base, head = prepare_git_repository(
-            workdir: Pathname(dir).realpath,
-            smoke_target: expectations.parent.join(params.name).realpath,
-            out: out,
-          )
+          repo_args = {
+              workdir: Pathname(dir).realpath,
+              smoke_target: expectations.parent.join(params.name).realpath,
+              out: out,
```

Comment:
[nitpick] Fix indentation.

```suggestion
            workdir: Pathname(dir).realpath,
            smoke_target: expectations.parent.join(params.name).realpath,
            out: out,
```

Reply:
fixed: 26cbef794e0cb9329b56970ab65195d47e44719d